src/Tasks/Resources/Strings.resx

```diff
@@ -1615,6 +1615,12 @@
         LOCALIZATION: Please don't localize "CopyLocal" this is an item meta-data name. "CopyLocalDependenciesWhenParentReferenceInGac" is a property name.
    </comment>
   </data>
+  <data name="ResolveAssemblyReference.NoBecauseBadImage">
+    <value>This reference is not "CopyLocal" because it is a bad image. It may be a native assembly, or it may not be an assembly at all.</value>
```

Comment:
I think "assembly" should only be used for .NET things.

```suggestion
    <value>This reference is not "CopyLocal" because it is a bad image. It may be a native binary, or it may not be an assembly at all.</value>
```